code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -170,15 +170,15 @@
 /obj/structure/particle_accelerator/particle_emitter/right/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
 	to_chat(S, "<span class='warning'>Disrupting the power grid would bring no benefit to us. Aborting.</span>")
 
-/obj/structure/particle_accelerator/particle_emitter/end_cap/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+/obj/structure/particle_accelerator/end_cap/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
```

Comment:
You'd think that would be the case, but it is not. Test it for yourself, swarmers can eat those two parts in the current code.